src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    public static final int FIGURE_COUNT = 5;

```

Comment:
If we don't use these constants outside of class it's recommended to make access modifier as strict as possible to follow encapsulation principle

Reply:
done